fth/floats.fth

```diff
@@ -23,6 +23,13 @@
 
 ANEW TASK-FLOATS.FTH
 
+: safe-flog
```

Comment:
Rather than make FLOG safe we should prevent the callers from making illegal calls to FLOG.